FinancialTracker/SceneDelegate.swift

```diff
@@ -46,8 +46,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                     window.rootViewController = navVC
                 }
             } else {
-                if let firebaseError = firebaseError {
-                    assertionFailure("Can't access user data \(firebaseError.localizedDescription)")
+                if let authError = authError {
+                    assertionFailure("Can't access user data \(authError.localizedDescription)")
```

Comment:
This either has to return an alert saying there is an `authError` of some kind or you should be using `fatalError` because in production leaving an empty return will do nothing and leave you with no information of what happened.